mars/services/task/execution/ray/executor.py

```diff
@@ -293,7 +303,27 @@ async def get_available_band_resources(self) -> Dict[BandType, Resource]:
 
     async def get_progress(self) -> float:
         """Get the execution progress."""
-        return 1
+        executor_progress = 0.0
+        assert len(self._stage_tile_progresses) == len(self._stage_output_object_refs)
```

Comment:
Any necessary to track all the output object refs of all stages?

Reply:
Not necessary. Just calculating current stage progress is ok. I'll optimize this.